code/handlers/trackservers.q

```diff
@@ -335,10 +335,11 @@ reqprocsnotconn:{[requiredprocs]
   }
 
 // Block process until all required processes are connected
-startupdependent:{[requiredprocs;timeintv]
-    while[.servers.reqprocsnotconn[requiredprocs];
-          .os.sleep[timeintv];
-          .servers.startup[]
+startupdependent:{[requiredprocs;timeintv;cycles;subscribe]
+     a:();
```

Comment:
a comment to explain the purpose of `a` would be good here, and possibly a more descriptive name. `n` often gets used for counters, might be more clear?